content/en/blog/2025/infernece-extension-support/index.md

```diff
@@ -0,0 +1,155 @@
+
+## **How Inference Routing Works in Istio**
+
+Istio's implementation builds on the service mesh's proven traffic management foundation. When a request enters the mesh through an Istio Gateway, it follows the standard Gateway API HTTPRoute matching rules. However, instead of using traditional load balancing algorithms, the backend is picked by an Endpoint Picker (EPP) service.
+
+The EPP evaluates multiple factors to select the optimal backend:
+
+**Request Criticality Assessment**: Critical requests receive priority routing to available servers, while lower criticality requests (Standard or Sheddable) may be load-shed during high utilization periods.
+
+**Resource Utilization Analysis**: The extension monitors GPU memory usage, particularly KV cache utilization, to avoid overwhelming servers that are approaching capacity limits.
+
+**Adapter Affinity**: For models using LoRA adapters, requests are preferentially routed to servers that already have the required adapter loaded, eliminating expensive loading overhead.
+
+**Prefix-Cache Aware Load Balancing**: Routing decisions consider distributed KV cache states across model servers, and prioritize model servers that already have the prefix in their cache.
+
+**Queue Depth Optimization**: By tracking request queue lengths across backends, the system avoids creating hotspots that would increase overall latency.
+
+This intelligent routing operates transparently within Istio's existing architecture, maintaining compatibility with features like mutual TLS, access policies, and distributed tracing.
+
+### **Inference Routing Request Flow**
+
+{{< image width="100%"
+    link="./inference-request-flow.svg"
+    alt="Flow of an inference request with gateway-api-inference-extension routing."
+    >}}
+
+## **The Road Ahead**
+
+**Support for Waypoints** - As Istio continues to evolve toward ambient mesh architecture, inference-aware routing will be integrated into waypoint proxies to provide centralized, scalable policy enforcement for AI workloads.
+
+Beyond Istio-specific innovations, the Gateway API Inference Extension community is also actively developing several advanced capabilities that will further enhance routing for for AI inference workloads on Kubernetes:
+
+**HPA Integration for AI Metrics**: Horizontal Pod Autoscaling based on model-specific metrics rather than just CPU and memory.
+
+**Multi-Modal Input Support**: Optimized routing for large multi-modal inputs and outputs (images, audio, video) with intelligent buffering and streaming capabilities.
+
+**Heterogeneous Accelerator Support**: Intelligent routing across different accelerator types (GPUs, TPUs, specialized AI chips) with latency and cost-aware load balancing.
+
+## **Getting Started with Istio Inference Extension**
+
+Ready to try inference-aware routing with Istio? The implementation is officially available starting release 1.27!
```

Comment:
```suggestion
Ready to try inference-aware routing? The implementation is officially available starting in Istio 1.27!
```

Reply:
"starting in" sounds weird, was it intentional?

Reply:
changed to "Istio 1.27"

Reply:
perhaps it's just an international thing, you could say "starting with", but it was missing a word wither way